sale_order_import/wizard/import_sale_view.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="import_sale_view_form" model="ir.ui.view">
+        <field name="name">import.sale.form</field>
+        <field name="model">import.sale</field>
+        <field name="arch" type="xml">
+            <form string="Import">
+                <separator string="Import Sale Orders" />
+                <span
+                >This wizard will create sales orders using imported .csv file. It will also validate sales order, validate invoice and pay the invoice.</span>
+                <group>
+                    <group>
+                        <field name="input_file" filename="datas_fname" />
+                        <field name="datas_fname" invisible="1" />
+                    </group>
+                    <group>
+                        <field name="asynchronous" />
+                        <field name="process_payment" />
+                    </group>
+                </group>
+                <group>
+                    <group>
+                        <field name="picking_policy" />
+                    </group>
+                </group>
+                <group>
+                    <group>
+                        <field
+                            name="customer_invoice_journal_id"
+                            domain="[('type', '=', 'sale')]"
+                        />
+                    </group>
+                    <group attrs="{'invisible':[('process_payment','=',False)]}">
+                        <field
+                            name="customer_payment_journal_id"
+                            domain="[('type', 'in', ('cash', 'bank'))]"
+                            attrs="{'required': [('process_payment', '=', True)]}"
+                        />
+                    </group>
+                </group>
+                <footer>
+                    <button
+                        name="import_sale_data"
+                        string="Import"
+                        type="object"
+                        class="oe_highlight"
+                    />
+                    or
+                    <button special="cancel" string="Cancel" class="oe_link" />
+                </footer>
+            </form>
+        </field>
+    </record>
+    <record id="import_sale_action" model="ir.actions.act_window">
+        <field name="name">Import Sale Order</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">import.sale</field>
+        <field name="view_mode">form</field>
+        <field name="view_id" ref="import_sale_view_form" />
+        <field name="context">{}</field>
+        <field name="target">new</field>
+    </record>
+    <menuitem
+        id="sale_import_wiz_menu"
+        name="Import Sale Order"
+        action="import_sale_action"
+        parent="base_data_import.data_import_main_menu"
+        sequence="10"
+    />
```

Comment:
I would prefer to keep the menu items inside the sales app, in a similar manner as https://github.com/qrtl/mi7-private/blob/2811440533ba3993601a468d9e184269dd2ac15c/stock_delivery_result_import/wizards/stock_delivery_result_import_views.xml#L30-L43.